src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -0,0 +1,42 @@
+import classNames from 'classnames';
+import React, { useState } from 'react';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const [isBtnVisible, setIsBtnVisible] = useState(false);
+
+  const { title, completed } = todo;
```

Comment:
destructurizing first
```suggestion
export const TodoInfo: React.FC<Props> = ({ todo }) => {
  const { title, completed } = todo;
  
  const [isBtnVisible, setIsBtnVisible] = useState(false);
```